site/content/contribute/developer-setup/_index.md

```diff
@@ -27,7 +27,7 @@ If you're using Windows, we recommend using the Windows Subsystem for Linux (WSL
 # Setup the Mattermost Server
 
 {{<note "Note:">}}
-The webapp isn't exposed directly, it's exposed via the server. So if both server and webapp are running, you can open `localhost:8065`, the server's port to access the webapp.
+The web app isn't exposed directly, it's exposed via the server. So if both server and web app are running, you can open `localhost:8065`, the server's port to access the web app.
```

Comment:
I feel like `webapp` is the canonical term, though I see we're using both in this repo. Should we stick with `webapp` to match the directory name?

Reply:
@hmhealey - I imagine you have a recommendation here?